ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Highlightings/FcsErrors.xml

```diff
@@ -292,9 +292,9 @@
   </Error>
 
   <Error staticGroup="FSharpErrors" name="NamespaceCannotContainValues">
-    <Parameter type="IFSharpIdentifier" name="identifier"/>
+    <Parameter type="IFSharpTreeNode" name="node"/>
```

Comment:
This change may make some things somewhat worse, since we can't offer better/different highlightings/texts for different cases. What do you think about making separate highlightings for bindings and expressions? That would allow having simpler and more correct texts like `Namespaces can't contain bindings`.